packages/terra-graphs/CHANGELOG.md

```diff
@@ -19,6 +19,6 @@
   * Added reflow to the Gantt component.
 
 * Changed
-  * Made changes to WDIO test url's according to terra rules.
+  * Made changes to WDIO test url's according to terra rules. 
```

Comment:
Is this change adding an extra space? If so, I would remove it.

Reply:
I made a test change in line graph, that is the reason why i added a space here.